### PR TITLE
hyperlinking contributors

### PR DESCRIPTION
**Pull request by adon-at-work**
as titled.
@neraliu , please review

### CODE REVIEW

File: tests/integration/run-secure-handlebars-spec.js [hunk 1]
```diff
@@ -90,7 +90,24 @@ describe("SecureHandlebars: compilation tests", function() {
...
+describe("SecureHandlebars: preprocess tests", function() {
+    it('smoke template', function(){
+        var html = '<!--{{hello}}--><a href={{url}} style="background:url({{url}})" id="{{id}}" id=\'{{id}}\'>{{hello}}</a>';
+        var data = {
+            'id': '12`"\'3',
+            'hello': 'hello',
+            'url': 'javascript:alert(1)'
+        };
+
+        var preprocessed = Handlebars.preprocess(html);
```
Comment (neraliu) [inline tests/integration/run-secure-handlebars-spec.js]: is the preprocess a new API?

Comment (adon-at-work) [inline tests/integration/run-secure-handlebars-spec.js]: not really new. it's been exposed quite some time, and is also tested in unit tests. this is to only enhance the coverage of integration test.

**Comment by adon-at-work** [comment]
@neraliu , any further comments? thanks.

**Comment by neraliu** [comment]
look good now

**Comment by adon-at-work** [comment]
merge?

**Comment by adon-at-work** [comment]
rebased after new master to resolve conflict. 
@neraliu

File: tests/integration/run-secure-handlebars-spec.js (hunk 1 above)
Comment (neraliu) [inline tests/integration/run-secure-handlebars-spec.js]: looks good now. the use case comes from the demo of SH

**Pull request merged**